posts/Modernize-your-pipeline-to-be-ARM-ready-with-multi-arch-container-builds.md

```diff
@@ -0,0 +1,233 @@
+---
+authors:
+  - John Blat, Lucas Ward
+tags:
+  - ARM
+  - Graviton
+  - DevOps
+  - Multi-arch
+  - ARM64
+  - aarch64
+  - amd64
+  - Ampere Altra
+  - Pipeline
+  - GitLab
+date: 2023-03-17T00:00:00.000Z
+title: "Modernize your pipeline to be ARM-ready with multi-arch container builds"
+---
+
+ARM computing has been historically reserved for embedded and small portable devices, however, in recent years there has been a growing number of ARM products in cloud and laptop products. This includes AWS Graviton Compute Instances, Raspberry Pi's Cortex-A72 chip, Googles Tau T2A compute instances, Azures Ampre Alta machines, and the pivot of Apple's ecosystem to ARM based processors. I know what you are thinking, Graviton, Tau, and Ampre Alta are really cool names, I agree! Given this new wave of ARM products, it is important to support the ARM architecture with multi-arch container images. Converting your pipeline to build multi-arch images is a relatively straightforward and quick process that pays off in a larger user base and higher cost savings. At Ippon Technologies, we utilize docker's buildx plugin to achieve building multi-arch images. Below is a short getting started guide, as well as some discussion as why, as a developer, it is in your best interest to build multi-arch images. 
```

Comment:
Also, "I know what you are thinking: Graviton, Tau, and Ampre Alta are really cool names. I agree!" un-run-ons this sentence.